app/src/main/java/com/randomappsinc/studentpicker/database/MySQLiteHelper.java

```diff
@@ -273,7 +273,7 @@ public void onUpgrade(SQLiteDatabase database, int oldVersion, int newVersion) {
             safelyAddColumnToTable(
                     database,
                     DatabaseColumns.NUMBER_OF_GROUPS,
-                    "INTEGER DEFAULT 1",
+                    "INTEGER DEFAULT 0",
```

Comment:
I feel like this should be 1? The code is resilient and doesn't crash if the settings are bogus I think.